linode/manager
# fix: [M3-6728] - Make drawers full width on mobile viewports

## Description 📝
Ensure that drawers are full width on mobile viewports

## Major Changes 🔄
- change drawer container to have width of 100% when viewport is less than the theme's 'sm' breakpoint (which is 600px). Added an additional maxWidth css property because there were some cases where the drawer just didn't look great with just `width: 100%` (for if viewport is ~450-600px, will show screenshots in comments), but can remove this if needed

## Preview 📷
(the orange strip on the after is just the laptop background)
| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/linode/manager/assets/139280159/9a784b60-9f7a-4d89-bb6b-7e814beca151) | ![image](https://github.com/linode/manager/assets/139280159/4e7ba639-8adb-4472-b138-ecd58aa883ee) |


## How to test 🧪
1. ensure that all drawers throughout site are full width on mobile viewports (caveat -- when the viewport is 450-600px, the drawer is not yet full width, will put up some screenshots) + ensure drawer functionality still works as expected
